pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -513,13 +513,20 @@ class PCPhase(Operation):
     r"""
     A projector-controlled phase gate.
 
+    This gate applies a complex phase :math:`\phi` to first :math:`dim`
```

Comment:
Thanks for adding this!

Reply:
+1! I think this is much more informative :+1: